bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -1072,22 +1083,23 @@ public void run() {
                 if (qe == null) { // no more queue entry
                     continue;
                 }
+                if (qe.entryId != Bookie.METAENTRY_ID_FORCE_LEDGER) {
```

Comment:
Who will be consuming METAENTRY_ID_FORCE_LEDGER??

Reply:
It is just a placeholder. No data needs to be written to journal.
forceLedger is using the same code path as addEntry but with this special id.